website/blog/2023-12-06-release-1.6.md

```diff
@@ -0,0 +1,250 @@
+hide_table_of_contents: false
+image: /img/blog/podman-desktop-release-1.6/SantaSeal.png
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.6 Release! 🎉
+
+![Podman-desktop-1-6-hero](img/podman-desktop-release-1.6/SantaSeal.png)
+
+This release introduces:
+- **Minikube featured extension**: Minikube extension to create local Kubernetes clusters in containers.
+- **Podman 4.8.2**: [Podman 4.8.2](https://github.com/containers/podman/releases) is now included in Windows and Mac installers.
+- **Setting page for Command-Line tools**: Manage and update your CLI tools.
+- **Kubernetes contexts Manager**: Browse all your kubernetes context, set default and remove unused ones.
+- **Editable Podman Machine**: Easy resize and reconfiguration of the Podman runtime environment.
+- **Filters for Containers and Pods lists**: Focus on the containers and Pods you are working with.
+- **Sorting on Volumes and Images list**: Sort volumes or images with your prefered criterias.
+- **Enhancement to show environment colums on Containers and Pods lists**: 
+- **Extension API improvements**: Another set of improvements to the extension API enabling more goodness from 🦭 Podman Desktop's extensions
+
+Podman Desktop 1.6 is now available. [Click here to download it](/downloads)!
+
+---
+
+## Release Details
+
+### Minikube featured extension
+For developers who need to run Kubernetes locally and reproduce an environment close to production for development and experimentation purposes, Podman Desktop allows users to easily set up that environment on a local machine. There are two distincts extensions providing the capability to configure a Kubernetes cluster locally, you can either choose between [Kind](https://kind.sigs.k8s.io/) or [minikube](https://minikube.sigs.k8s.io/docs/).
+
+The minikube extension allows to setup a Kubernetes cluster locally running in a container! Yeah, you read well, in a container, in a similar fashion than Kind. The advantage is that it's lighter and faster to start. With minikube, one of the advantage, is that you can build your images locally with podman and get them automatically available in your local Kubernetes cluster -  which will speed up your turnarounds when you want to test your application. If you want to learn more about the [following blog post](https://podman-desktop.io/blog/sharing-podman-images-with-kubernetes-cluster).
+
+![Minikube-feature-extension](img/podman-desktop-release-1.6/minikube-feature-extension.png)
+
+
+### Command-Line Tools Configuration: Compose and Kubectl
+Configuring and managing your setup is getting easier with the addition of a new section in the Settings to manage Command-line tools. In Podman Desktop, extensions can bring dependencies and tools that are going to be necessary for working properly. 
+
+Two Command-Line tools are getting registered at the moment Compose and kubectl:
+- Compose-cli is registered by the Compose extension. 
```

Comment:
Both of these feel like they're listing extension dependencies (ok, they kind of are...) vs providing easier ways to install and keep these clis up to date. It would sound better to reverse this to talk about what the cli does/the benefit and the extension is more of a footnote. e.g.:

- Compose-cli for running 'podman compose' commands (provided by the Compose extension).
- kubectl for interacting with Kubernetes clusters (provided by the minikube extension).